From 4e50609edf5c38dcb6fb979b2d5dd2448ebe3e31 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Sat, 19 Jul 2014 03:31:01 +1000
Subject: [PATCH 2/3] metadata: also look for artwork when scanning a single
 recording

no point waiting a day or having to run a full artwork scan
---
 mythtv/programs/mythmetadatalookup/lookup.cpp |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/mythtv/programs/mythmetadatalookup/lookup.cpp b/mythtv/programs/mythmetadatalookup/lookup.cpp
index e373f84..ba818be 100644
--- a/mythtv/programs/mythmetadatalookup/lookup.cpp
+++ b/mythtv/programs/mythmetadatalookup/lookup.cpp
@@ -50,9 +50,10 @@ void LookerUpper::HandleSingleRecording(const uint chanid,
     }
 
     m_updaterules = updaterules;
+    m_updateartwork = true;
 
     m_busyRecList.append(pginfo);
-    m_metadataFactory->Lookup(pginfo, true, false, false);
+    m_metadataFactory->Lookup(pginfo, true, m_updateartwork, false);
 }
 
 void LookerUpper::HandleAllRecordings(bool updaterules)
-- 
1.7.10.2

